### PR TITLE
add service-mesh docs

### PR DESCRIPTION
**Pull request by zackzhangkai**
Signed-off-by: zackzhangkai <zackzhang@yunify.com>

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
@zackzhangkai The images were missing in the previewing page: https://deploy-preview-287--kubesphere-v3.netlify.app/docs/project-user-guide/grayscale-release/blue-green-deployment/

File: content/en/docs/project-user-guide/grayscale-release/blue-green-deployment.md [hunk 1]
```diff
@@ -7,4 +7,77 @@ linkTitle: "Blue-green Deployment"
...
+![](./../../../../../static/images/service-mesh/istio-1.jpg)
+
+
+Secondly, you should set project gateway to enable application governance, in the project advanced settings.
+
+![](./../../../../../static/images/service-mesh/application-1.jpg)
+
+Thirdly, you should have an application, such as bookinfo sample as follows.
+
+![](./../../../../../static/images/service-mesh/application-2.jpg)
```
Comment (FeynmanZhou) [inline content/en/docs/project-user-guide/grayscale-release/blue-green-deployment.md:32]: Replace this section with this [doc](https://kubesphere.io/docs/pluggable-components/service-mesh/#enable-service-mesh-before-installation).

File: content/en/docs/project-user-guide/grayscale-release/blue-green-deployment.md [hunk 2]
```diff
@@ -7,4 +7,77 @@ linkTitle: "Blue-green Deployment"
...
+
+
+## Before you begin
+
+Firstly, you shoud enable the service-mesh plugin, to make the istio component available.
+
+```
+kubectl edit cc ks-installer -n kubesphere-system
+```
+![](./../../../../../static/images/service-mesh/istio-1.jpg)
```
Comment (FeynmanZhou) [inline content/en/docs/project-user-guide/grayscale-release/blue-green-deployment.md:23]: All image URLs are mistaken.

File: content/en/docs/project-user-guide/grayscale-release/blue-green-deployment.md [hunk 3]
```diff
@@ -7,4 +7,77 @@ linkTitle: "Blue-green Deployment"
...
 
-TBD
\ No newline at end of file
+
+The blue-green release provides a zero downtime deployment, which means the new version can be deployed with the old one preserved. At any time, only one of the version is active serving all the traffic, while the other one remains idle. If there is a problem with running, you can quickly roll back to the old version
+
+![](/images/service-mesh/blue-green-0.png)
+
+
+## Before you begin
```
Comment (FeynmanZhou) [inline content/en/docs/project-user-guide/grayscale-release/blue-green-deployment.md:16]: Replace the steps of the section **Before you begin** with this [doc](https://kubesphere.io/docs/pluggable-components/service-mesh/#enable-service-mesh-before-installation), you just need to add a reference link to it.

**Comment by Sherlock113** [comment]
/lgtm
This pr can be merged and I will update wording later.

**Review by rayzhou2017 — APPROVED** [review]
/lgtm

**Pull request merged**